docs/quickstart_guide/index.rst

```diff
@@ -0,0 +1,245 @@
+.. _quickstart_guide:
+
+Quickstart Guide
+################
+
+This section aims to get you started with Lewis as quickly as possible. It is meant as a basic starting point for becoming familiar with Lewis, and to give you a broad overview of what it can do. As such, many features are skimmed over or skipped entirely. See the detailed documentation sections for a more complete overview of features.
+
+This guide is presented as a step-by-step tutorial, so skipping sections may mean you will miss steps that are required for the examples to work.
+
+
+Install Lewis
+=============
+
+The recommended way to install Lewis is via PyPI and using a virtual environment. This guide assumes you have Python and Pip installed and in your PATH.
+
+Install ``virtualenv`` and create a virtual environment. On Windows, the activate script will be located elsewhere and can be executed directly, without ``source``. This step is optional and may be skipped:
+
+::
+
+    $ pip install virtualenv
+    $ virtualenv myenv
+    $ source myenv/bin/activate
+
+Lewis can be installed with Pip using a single command:
+
+::
+
+    $ pip install lewis
+    $ lewis --version
+
+
+Run the Motor Example
+=====================
+
+Once Lewis is installed, you can use it to start some of the example devices it ships with. 
+
+You can see which devices are available by just executing Lewis without parameters:
+
+::
+
+    $ lewis
+    Please specify a device to simulate. The following devices are available:
+        julabo
+        chopper
+        linkam_t95
+
+Some additional, simpler examples are located in the ``lewis.examples`` module. You can tell Lewis which module to scan for devices using the ``-k`` parameter:
+
+::
+
+    $ lewis -k lewis.examples
+    Please specify a device to simulate. The following devices are available:
+        dual_device
+        simple_device
+        timeout_device
+        modbus_device
+        example_motor
+
+For this guide, we will launch the example_motor:
+
+::
+
+    $ lewis -r :10000 -k lewis.examples example_motor
+    INFO lewis.DeviceBase: Creating device, setting up state machine
+    INFO lewis.Simulation: Changed cycle delay to 0.1
+    INFO lewis.Simulation: Changed speed to 1.0
+    INFO lewis.Simulation: Starting simulation
+    INFO lewis.ControlServer: Listening on 0.0.0.0:10000
+    INFO lewis.AdapterCollection: Connecting device interface for protocol 'stream'
+    INFO lewis.ExampleMotorStreamInterface.StreamServer: Listening on 0.0.0.0:9999
+
+The example motor is a TCP Stream device and listens on port 9999 on all adapters by default.
+
+The ``-r`` parameter is optional and specifies an ``adapter:port`` pair for the "Control API" to listen on. Here it listens on port 10000 on all adapters. We will see more on that in following sections.
```

Comment:
Maybe start the sentence by saying "you can ignore the `-r` parameter at this point". and then continue to say what it refers to.

Reply:
As you don't need the control server immediately. Can we delay introducing it?

Reply:
Yeah, I should probably just remove it here.

I think my idea was to avoid having to restart Lewis later, but I end up explaining how to restart it with remote access anyway.